docs/source/index.mdx

```diff
@@ -4,13 +4,14 @@ title: Introduction to Apollo Server
 
 **Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](https://www.apollographql.com/docs/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-![diagram](./images/index-diagram.svg)
+<img src="./images/frontend_backend_diagram.svg" width="500" alt="Diagram showing Apollo Server bridging frontend and backend" />
 
 #### You can use Apollo Server as:
 
+* A gateway for a [federated supergraph](/federation/)
+  * The GraphQL server for a [subgraph](/federation/subgraphs) in a federated supergraph
```

Comment:
@rkoron007  Was this intended to be more deeply nested?

Reply:
This was meant to be nested, but on second look I think I would like these all to be on the same level. I'll remove that indent. :)